pkg/client/events/errors.go

```diff
@@ -0,0 +1,14 @@
+package events
+
+import (
+	sdkerrors "cosmossdk.io/errors"
+)
+
+var (
+	codespace = "events"
+
+	ErrDial                 = sdkerrors.Register(codespace, 1, "dialing for connection failed")
+	ErrConnClosed           = sdkerrors.Register(codespace, 2, "connection closed")
+	ErrSubscribe            = sdkerrors.Register(codespace, 3, "failed to subscribe to events")
+	ErrEventsUnmarshalEvent = sdkerrors.Register(codespace, 4, "failed to unmarshal event bytes")
```

Comment:
Again this is my fault, but wdyt about renaming these to be more consistent with the naming structure our on-chain errors?